Xamarin.Forms.Core/CornerRadius.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Diagnostics;
+
+namespace Xamarin.Forms
+{
+	[DebuggerDisplay ("TopLeft={TopLeft}, TopRight={TopRight}, BottomLeft={BottomLeft}, BottomRight={BottomRight}")]
+	[TypeConverter (typeof (CornerRadiusTypeConverter))]
+	public struct CornerRadius
+	{
+		public double TopLeft { get; private set; }
+		public double TopRight { get; private set; }
+		public double BottomLeft { get; private set; }
+		public double BottomRight { get; private set; }
+
+		internal bool IsDefault
```

Comment:
no need to expose this. it's not more expensive to do `if (cornerRadius == default(CornerRadius))`

Reply:
Do you mean the property IsDefault?